src/models/message_status.ts

```diff
@@ -57,8 +86,21 @@ export const RetrievedMessageStatus = wrapWithKind(
 
 export type RetrievedMessageStatus = t.TypeOf<typeof RetrievedMessageStatus>;
 
+// --------------------------------------
+// MessageStatusUpdater
+// --------------------------------------
+
 export type MessageStatusUpdater = (
-  status: MessageStatusValueEnum
+  statusUpdate:
```

Comment:
Why not create a type with the union of these types?

Reply:
Thanks, done in [222e6fb](https://github.com/pagopa/io-functions-commons/pull/289/commits/222e6fbd487062196b152d9f53f9839a0eb949fd)